src/calculateRentalCost.js

```diff
@@ -5,6 +5,7 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  return days < 3 ? days * 40 : days < 7 ? days * 40 - 20 : days * 40 - 50;
```

Comment:
[CheckStyle] Don't use literals in expressions (HARDCODE). Instead use constants explaining this values
BAD EXAMPLE:
if (numberOfDays >= 7) {
  return basePrice - 50;
}
GOOD EXAMPLE:
const LONG_TERM = 7;
const LONG_TERM_DISCOUNT = 50;

if (numberOfDays >= LONG_TERM) {
  return basePrice - LONG_TERM_DISCOUNT;
}